cadctap/setup.cfg

```diff
@@ -36,7 +36,7 @@ edit_on_github = False
 github_project = opencadc/caom2tools 
 # version should be PEP386 compatible (http://www.python.org/dev/peps/pep-0386)
 install_requires = cadcutils>=1.1.15 clint>=0.5.1 python-magic>=0.4.15 requests>=2.18 requests-toolbelt>=0.8.0
-version = 0.8
+version = 0.9
```

Comment:
Make the version 0.8.1 since it doesn't introduce major API changes and it's mostly bug fixes.

Reply:
Done